UT4MasterServer/Controllers/SessionController.cs

```diff
@@ -37,118 +38,112 @@ public class SessionController : JsonAPIController
 	[AuthorizeBasic]
 	[HttpPost("token")]
 	public async Task<IActionResult> Authenticate(
-		[FromForm(Name = "grant_type")] string grantType,
-		[FromForm(Name = "includePerms")] bool? includePerms,
-		[FromForm(Name = "code")] string? code,
-		[FromForm(Name = "exchange_code")] string? exchangeCode,
-		[FromForm(Name = "refresh_token")] string? refreshToken,
-		[FromForm(Name = "username")] string? username,
-		[FromForm(Name = "password")] string? password)
+		[FromBody] AuthenticateRequest request)
```

Comment:
These changes may break the old login/register pages, I haven't tested this, but it shouldn't matter as long as you guys can get the new to work.